sov-modules/sov-app-template/src/lib.rs

```diff
@@ -263,6 +216,114 @@ where
             inner: SequencerOutcome::Rewarded,
         }
     }
+}
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
```

Comment:
Not relevant to this PR but we should feature gate `serde`

Reply:
Do we have feature for that? Because this feature-gate is inconsistent across codebase